projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,29 +4,26 @@
 
 ## 11.3-a.1 - 2022-08-03
 ### Enhancements
-- Add blocks for GSuite document embeds. [#24628]
-- Blocks: add support for podcast player to have per-feed cache timeouts. [#24966]
-- Include 'earn' patterns in the Payment Introducer block UI [#24914]
-- Jetpack: register VideoPress block from its editor.js file [#25349]
-- VPBlock: Add support to pause/resume upload [#25350]
+- GSuite blocks (beta): add blocks for Google document embeds. [#24628]
```

Comment:
Not sure if this is the best way to name three GSuite blocks. Also they appear to be available on WP.com only at the moment?

Reply:
They likely can't be used on Jetpack / WoA unless the Beta blocks constant is enabled. It could be written similarly to how the Recipe block was introduced here: https://github.com/Automattic/jetpack-production/releases/tag/11.1

Reply:
```suggestion
- GSuite blocks (beta): add blocks for Google document embeds. Currently a JETPACK_BETA_BLOCKS feature. [#24628]
```

@coder-karen Like this?

Reply:
Looks good.

Reply:
Might want to add the same for the readme.txt though.